laske_export/management/commands/send_invoices_to_laske.py

```diff
@@ -66,14 +67,39 @@ def handle(self, *args, **options):
                 )
             )
 
-            email_content = _("MVJ ({}) sent {} invoices to Laske on {}").format(
+            export_invoice_log_items = (
+                laske_export_log_entry.laskeexportloginvoiceitem_set
+            )
+            sent_count = export_invoice_log_items.filter(
+                status=LaskeExportLogInvoiceStatus.SENT
+            ).count()
+            failed_count = export_invoice_log_items.filter(
+                status=LaskeExportLogInvoiceStatus.FAILED
+            ).count()
+
+            email_content = _(
+                "MVJ ({}) processed a total of {} invoices to Laske system on {}. "
+                "Of the invoices, {} succeeded and {} failed."
+            ).format(
                 settings.LASKE_VALUES["sender_id"],
                 laske_export_log_entry.invoices.count(),
                 laske_export_log_entry.ended_at.astimezone(
                     timezone.get_current_timezone()
                 ).strftime("%d.%m.%Y %H.%M %Z"),
+                sent_count,
+                export_invoice_log_items.filter(
+                    status=LaskeExportLogInvoiceStatus.SENT
+                ).count(),
             )
 
+            if failed_count > 0:
+                email_content += "\n\n"
+                email_content += _("Failed invoice numbers:") + ""
+                for invoice in laske_export_log_entry.invoices.filter(
+                    laskeexportloginvoiceitem__status=LaskeExportLogInvoiceStatus.FAILED
+                ):
+                    email_content += "\n* {}".format(invoice.number)
```

Comment:
I don't think there is an easy way to find an invoice by its number in the UI, so I'd like to add the lease identifier to this line so it will be easier for the user to find the offending invoice.

Something like `"\n* #{} ({})".format(invoice.number, str(invoice.lease.identifier))`

Reply:
Nice, this has been changed for your suggestion.